.try {
leave L_1
}
catch class [mscorlib]System.Exception {
.locals init (class [mscorlib]System.Exception 'ex.0')
stloc 'ex.0'
leave L_1
}
catch class 'Test'.'MyException' {
.locals init (class 'Test'.'MyException' 'ex.1')
stloc 'ex.1'
leave L_1
}
L_1:
